apps/web/templates/ascents/_form.html.erb

```diff
@@ -18,13 +18,13 @@
 
       div class: 'input form-group' do
         label  :kind
-        select :kind, kinds, class: 'form-control'
+        select :kind, Hash['line' => 'line', 'boulder' => 'boulder']
       end
     end
 
     div class: 'input form-group' do
       label  :style
-      select :style, styles, class: 'form-control'
+      select :style, Hash['os' => 'os', 'fl' => 'fl', 'rp' => 'rp']
```

Comment:
In my opinion, we should't hardecode nothing in the templates. If all the information about one thing (in this case styles), we relate to a one constant, then in the future if we would like to add new value(style), just add it to the constant. (maybe this is not the best example, because adding a new style is unlikely,  but it's about principles)

Reply:
The same: we don't yet know what the Style is in the context of our application. In my opinion, it doesn't belong to Ascent, it should be a separate entity that doesn't exist now. It's better to make things simple now, and it will be easier to refactor it later when other requirements come then to stick to solutions made in the very early development stage.

To sum up: I don't think we will stick to neither of the solutions, but this PR makes a better starting point for future changes. If you think otherwise, please feel free to delete this PR 👍